pkg/network/node/egressip.go

```diff
@@ -185,18 +180,23 @@ func (eip *egressIPWatcher) releaseEgressIP(egressIP, mark string) error {
 		return nil
 	}
 
-	localEgressIPMaskLen, _ := eip.localEgressNet.Mask.Size()
+	localEgressLink, localEgressNet, err := GetLinkDetails(eip.localIP)
```

Comment:
What about making this `GetLinkDetails(egressIP)`, so it works in all cases. Thoughts, @danwinship?